sysml.library/Kernel Library/Occurrences.kerml

```diff
@@ -150,6 +154,8 @@ package Occurrences {
 	assoc HappensBefore {
```

Comment:
I forgot to edit the second sentence of the HappensBefore comment (and add a third) to align with the updated HappensDuring comment above:
"That is, the time interval of the earlierOccurrence is completely before that of the laterOccurrence, or every snapshot of the earlierOccurrence happens before every snapshot of the laterOccurrence. Note that this means no occurrence happens before itself."

Reply:
Updated.